### ISSUE TITLE
Upload the missing NFT metadata in our servers to patch the missing `wildhorn_v2` NFTs.

### ISSUE THREAD

**Issue by jjperezaguinaga** [hoprnet/hopr-devrel#87]

**Comment by jjperezaguinaga** [comment]
Step 1 - Save all the specific arts into a folder as `$boostType/$boostRank`
Step 2 - Upload this folder and name it as `$boostType` into IPFS Pinata

![image](https://user-images.githubusercontent.com/1128312/140343896-79188d01-9fc0-414a-a837-f1dd8aeecfdd.png)

![image](https://user-images.githubusercontent.com/1128312/140344109-afa7bda8-b36d-4d66-adac-4366b117ab65.png)

**Comment by jjperezaguinaga** [comment]
Step 3 - Create a folder named `$boostType` in https://github.com/hoprnet/hopr-devrel/tree/main/stake/packages/frontend/public.
Step 4 - Create inside that folder, `$boostRank` files with the metadata for each as JSON. Make sure to update `type`, `rank`, `image`, and `boost` accordingly.

```
{
    "type": "Wildhorn_v2",
    "rank": "bronze",
    "image": "ipfs://QmUdi8CTTmgpj8DrKh2yzoeyb6pVEFKgV3hjaTPmG8Bzn3/bronze.png",
    "deadline": 1642424400,
    "boost": 0.03
}
```

Step 5 - Create a PR and request approval. Merge after approval and see it working with a user in https://stake.hoprnet.org

e.g. https://github.com/hoprnet/hopr-devrel/pull/88

**Comment by jjperezaguinaga** [comment]
Step 5 - Upon merge, make sure to also merge against `release/stake` otherwise production won't be updated. See #89 for reference.

**Comment by jjperezaguinaga** [comment]
Fixed in #89
